installation/CanvasVault/setup_CanvasVault.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+
+
+export AUTH_PATH="jwt-k8s-sman"
+export VAULT_DEV_ROOT_TOKEN_ID="egalegal"
+# optional create istio ingress virtualservice when hostname is set
+export CANVASVAULT_VS_HOSTNAME="canvas-vault-hc.ihc-dt.cluster-3.de"
+
+
+
+
+set -e
+
+cd $(dirname -- $0)
+
+
+Y='\033[0;33m'
+NC='\033[0m' # No Color
+
+TARGETENV="$1"
+if [ -z "$TARGETENV" ]; then
+	echo "COMMAND [AWS|GCP|VPS]"
```

Comment:
Do we want to support the target "VPS"?

Reply:
Yes. VPS deployment still works and it´s configuration differes from the other two. So, no need to remove it.